docs/rules/use-class-secure-random.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Use Class SecureRandom
+ruleId: UseClassSecureRandom
+since: 3.20.0
+minJavaVersion: 1.2
+remediationCost: 5
+links:
+    - displayName: "S2245"
+      url: "https://sonarcloud.io/organizations/default/rules?open=java%3AS2245&q=pseudorandom"
+    
+description:
+    This rule prevents vulnerabilities due to a pseudo-random number generator (PRNGs) by replacing it by a cryptographically strong random number generator (RNG).
+tags: ["Java 1.2", "Security"]
+---
+
+# Use Class SecureRandom
+
+[[toc]]
+
+## Properties
+
+<RuleProperties />
+
+## Description
+Because the class [java.util.Random](https://docs.oracle.com/javase/8/docs/api/java/util/Random.html) relies on a pseudo-random number generator, it should not be used for security-critical applications. An attacker might be able to guess the next value generated by a pseudo-random number generator and thus access sensitive data.
```

Comment:
Please make sure to use the `monospace code font` for keywords etc. 
A few example replacements:
[java.util.Random](https://docs.oracle.com/javase/8/docs/api/java/util/Random.html) -> [`java.util.Random`](https://docs.oracle.com/javase/8/docs/api/java/util/Random.html)
Random() -> `Random()`